diff -burN mecab-0.79.orig/dic/Makefile.in mecab-0.79/dic/Makefile.in
--- mecab-0.79.orig/dic/Makefile.in	2004-09-12 15:27:11.195109848 +0200
+++ mecab-0.79/dic/Makefile.in	2004-09-12 15:40:39.378247336 +0200
@@ -97,7 +97,7 @@
 install_sh = @install_sh@
 sort = @SORT@
 uniq = @UNIQ@
-dicdir = @prefix@/lib/mecab/dic/ipadic
+dicdir = @libdir@/mecab/dic/ipadic
 charset = @CHARSET@
 dic_DATA = dic.csv connect.csv da.me matrix.me feature.me info.me token.me dicrc
 CLEANFILES = *.me dic.csv connect.csv
