--- mecab-0.76/src/dictionary_maker.cpp.orig	2003-06-29 21:39:30.000000000 +0200
+++ mecab-0.76/src/dictionary_maker.cpp	2003-10-24 01:00:17.924228480 +0200
@@ -32,7 +32,7 @@
   {
     std::vector <unsigned int> result;
     getIdList (csv, result);
-    return result [result.size()-1];
+    return result.size() ? (result [result.size()-1]) : 0;
   }
 
   unsigned int DictionaryMaker::getIdList (Csv &csv, 
